Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/qrcode/RapidAntigenQrCodeExtractor.kt

```diff
@@ -49,7 +49,12 @@ class RapidAntigenQrCodeExtractor @Inject constructor() : QrCodeExtractor<Corona
         } else {
             BaseEncoding.base64Url().decode(this)
         }
-        return Gson().fromJson(decoded.commonToUtf8String())
+        try {
+            return Gson().fromJson(decoded.commonToUtf8String())
```

Comment:
Can `BaseEncoding.base64().decode(this)` also throw an exception?

Reply:
Hm I think it can, should catch that too?

```
  /**
   * Decodes the specified character sequence, and returns the resulting {@code byte[]}. This is the
   * inverse operation to {@link #encode(byte[])}.
   *
   * @throws IllegalArgumentException if the input is not a valid encoded string according to this
   *     encoding.
   */
  public final byte[] decode(CharSequence chars) {
```